lib/git_reflow/git_server/pull_request.rb

```diff
@@ -169,41 +169,36 @@ def deliver?
 
       def cleanup_failure_message
         GitReflow.say "Cleanup halted.  Local changes were not pushed to remote repo.".colorize(:red)
-        GitReflow.say "To reset and go back to your branch run \`git reset --hard origin/#{base_branch_name} && git checkout #{feature_branch_name}\`"
+        GitReflow.say "To reset and go back to your branch run \`git reset --hard origin/#{self.base_branch_name} && git checkout #{self.feature_branch_name}\`"
       end
 
       def merge!(options = {})
         if deliver? 
 
-          GitReflow.say "Merging pull request ##{number}: '#{title}', from '#{feature_branch_name}' into '#{base_branch_name}'", :notice
+          GitReflow.say "Merging pull request ##{self.number}: '#{self.title}', from '#{self.feature_branch_name}' into '#{self.base_branch_name}'", :notice
 
-          unless base_branch_name.index(':').nil?
-            base = base_branch_name[(base_branch_name.index(':') + 1)..-1]
-          end
-
-          unless feature_branch_name.index(':').nil?
-            feature = feature_branch_name[(feature_branch_name.index(':') + 1)..-1]
-          end
+          self.base_branch_name = self.base_branch_name[/[^:]+$/]
```

Comment:
@codenamev Line 180 is the same logic as what I am trying to do previously in line 180 - 186. This is just a bit more elegant Ruby Regex Syntax to match the expression in the string after the ":".

There are 2 cases: 
"master" => "master"
"codenamev:master" => "master"

Regex supports both ^_^